app/searches/locations_search.rb

```diff
@@ -34,6 +34,53 @@ def search
     search_results
   end
 
+  def exact_match_found?
+    keyword_to_use = keywords || attributes[:keyword]
+
+    if keyword_to_use.blank? && matched_category.blank?
+      return false
+    end
+
+    if matched_category.present?
+      return true
+    end
+
+    return false if keyword_to_use.blank?
+
+    exact_match_query = index.query(bool: {
+      should: [
+        { term: { "organization_name_exact": { value: keyword_to_use.downcase } } },
+        { term: { "name_exact": { value: keyword_to_use.downcase } } },
+        { term: { "categories_exact": { value: keyword_to_use.downcase } } },
+        { term: { "sub_categories_exact": { value: keyword_to_use.downcase } } },
+        { match_phrase: { "organization_name": { query: keyword_to_use, slop: 0 } } },
+        { match_phrase: { "name": { query: keyword_to_use, slop: 0 } } },
+        { match_phrase: { "categories": { query: keyword_to_use, slop: 0 } } },
+        { match_phrase: { "sub_categories": { query: keyword_to_use, slop: 0 } } }
+      ]
+    })
```

Comment:
I think this is trying to find the match like 1 - 1: 
For example if we look for "Baby Food" and one of those fields is "We have baby food available" or "Baby Food for all" it wont be found, plus maybe we want to expand the exact match to all the fields, specially descriptions. I found this stack overflow question that might help: 
https://stackoverflow.com/questions/22093334/how-to-make-query-string-search-exact-phrase-in-elasticsearch

Reply:
I tried to search for 'COVID Testing' and I got the banner and I saw on some of the descriptions the match for "COVID testing":

<img width="1451" alt="image" src="https://github.com/user-attachments/assets/0100f34f-ffa9-4ed3-8227-5ff2bba591db">

<img width="1200" alt="image" src="https://github.com/user-attachments/assets/02e7786c-ea35-422c-b68a-b507ab55150f">

Reply:
The partial matching is below that code. 
If you try to find "Medicine" in the keyword search, "Johns Hopkins Medicine" will be returned as a partial result match, even though it isn't 1-1

```
    # not an exact match, check for partial matches
    if !result
      partial_match_query = index.query(bool: {
        should: [
          { match: { "organization_name": { query: keyword_to_use, operator: "and" } } },
          { match: { "name": { query: keyword_to_use, operator: "and" } } },
          { match: { "categories": { query: keyword_to_use, operator: "and" } } },
          { match: { "sub_categories": { query: keyword_to_use, operator: "and" } } }
        ]
      })

      partial_results = partial_match_query.count
      result = partial_results > 0
    end
```